### PR TITLE
add apply-cache

### PR DESCRIPTION
**Pull request by Joyinqin**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/5892
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: system-variables.md [hunk 1]
```diff
@@ -791,6 +791,13 @@ For a system upgraded to v5.0.0-rc from an earlier version, if you have not modi
 - This variable is used to set the threshold value of memory quota for a query.
 - If the memory quota of a query during execution exceeds the threshold value, TiDB performs the operation designated by the OOMAction option in the configuration file. The initial value of this variable is configured by [`mem-quota-query`](/tidb-configuration-file.md#mem-quota-query).
 
+### tidb_mem_quota_apply_cache
+
+- Scope: SESSION | GLOBAL
+- Default value: 32 MB
+- This variable is used to set the memory usage threshold of the local Cache in the `Apply` operator.
```
Comment (ran-huang) [inline system-variables.md:798]: I don't think `Cache` should start with uppercase. WDYT @TomShawn

**Review by ran-huang — APPROVED** [review]
Rest lgtm

File: system-variables.md (hunk 1 above)
Comment (TomShawn) [reply system-variables.md:798]: I think you are right.

File: system-variables.md [hunk 2]
```diff
@@ -791,6 +791,13 @@ For a system upgraded to v5.0.0-rc from an earlier version, if you have not modi
...
+- The local Cache in the `Apply` operator is used to speed up the computation of the `Apply` operator. You can set the variable to `0` to disable the `Apply` Cache feature.
```
Comment (TomShawn) [inline system-variables.md:799]:
```suggestion
- The local cache in the `Apply` operator is used to speed up the computation of the `Apply` operator. You can set the variable to `0` to disable the `Apply` cache feature.
```

**Pull request merged**